### PR TITLE
Fix/command cleanup

### PR DESCRIPTION
**Pull request by surajgour-d11**
# Odin version(1.2.0-alpha)

## Features

- removes/modifies flags and commands as per the below doc
- added spinner support for output streams
- yml -> json outputs
## Resources

https://dream11.atlassian.net/wiki/spaces/COREINFRA/pages/2672459806/Command+CleanUp

### CODE REVIEW

File: internal/command/commands/env.go [hunk 1]
```diff
@@ -205,81 +153,39 @@ func (e *Env) Run(args []string) int {
 			return 1
 		}
 
-		tableHeaders := []string{"Name", "Team", "Env Type", "State", "Account", "Deletion Time", "Purpose", "CreatedAt", "CreatedBy", "UpdatedAt", "UpdatedBy"}
+		tableHeaders := []string{"Name", "Team", "Env Type", "State", "Account", "Deletion Time"}
```
Comment (akshaypatidar1999) [inline internal/command/commands/env.go:156]: Why createdBy, updatedBy etc are removed.
Can't find in the doc

File: pkg/sse/stream_events.go [hunk 2]
```diff
@@ -60,9 +63,25 @@ func (sr *StreamRequest) Stream() StreamResponse {
...
-		logger.Output(string(line))
+		line := string(data.Bytes())
+		if line == "" {
+			continue
+		}
+		if strings.Contains(line, ui.SPINNER) {
+			parts := strings.Split(line, ui.SPINNER)
+			s.Prefix = parts[0]
+			s.Suffix = parts[1]
+			err := s.Color("fgHiBlue", "bold")
```
Comment (akshaypatidar1999) [inline pkg/sse/stream_events.go:76]: Use this color from constants instead of hardcoding here

**Review by BalkishanNanavath — APPROVED** [review]

File: internal/command/commands/env.go (hunk 1 above)
Comment (surajgour-d11) [reply internal/command/commands/env.go:156]: In a discussion with Rahul, requirement was to reduce the size of table to simplify display. So, we decided to trim the table by removing these cols.

File: pkg/sse/stream_events.go (hunk 2 above)
Comment (surajgour-d11) [reply pkg/sse/stream_events.go]: done

File: internal/command/commands/env.go (hunk 1 above)
Comment (akshaypatidar1999) [reply internal/command/commands/env.go:156]: I think we should atleast have createdBy column

**Review by akshaypatidar1999 — DISMISSED** [review]

File: pkg/sse/stream_events.go [hunk 3]
```diff
@@ -60,9 +65,25 @@ func (sr *StreamRequest) Stream() StreamResponse {
...
-		logger.Output(string(line))
+		line := string(data.Bytes())
+		if line == "" {
+			continue
+		}
+		if strings.Contains(line, ui.SPINNER) {
+			parts := strings.Split(line, ui.SPINNER)
+			s.Prefix = parts[0]
+			s.Suffix = parts[1]
+			err := s.Color(SPINNER_COLOR, "bold")
```
Comment (BalkishanNanavath) [inline pkg/sse/stream_events.go:78]: we can create constant for bold.

File: internal/command/commands/env.go (hunk 1 above)
Comment (surajgour-d11) [reply internal/command/commands/env.go:156]: This is not decided yet. Will modify if required in later stages depending upon discussion.

File: pkg/sse/stream_events.go [hunk 4]
```diff
@@ -60,9 +65,25 @@ func (sr *StreamRequest) Stream() StreamResponse {
 	}
 
 	data := bufio.NewScanner(resp.Body)
+	s := spinner.New(spinner.CharSets[14], 100*time.Millisecond)
```
Comment (BalkishanNanavath) [inline pkg/sse/stream_events.go:68]: constants for charset and milliseconds

**Review by BalkishanNanavath — APPROVED** [review]

**Pull request merged**